ui/perfherder/perf-helpers/perfdocs.js

```diff
@@ -168,22 +176,37 @@ export class Perfdocs {
       frameworkName === 'performance-tests-overview'
         ? baseURL.concat('devtools/tests/')
         : baseURL.concat('testing/perfdocs/');
-    if (this.suite in invertedTestsNamesDevTools) {
-      this.suite = invertedTestsNamesDevTools[this.suite];
-    }
-    url = url.concat(
-      frameworkName,
-      '.html#',
-      this.suite.replace(/:|_|\s|\./g, '-').toLowerCase(),
-    );
+
+    url = url.concat(frameworkName, '.html#');
+
     if (frameworkName === 'raptor') {
-      if (browsertimeBenchmarks.includes(this.suite)) {
-        url = url.concat('-b');
-      } else if (browsertimeCustomTests.includes(this.suite)) {
-        url = url.concat('-c');
-      } else if (this.platform.includes('android')) {
-        url = url.concat('-m');
-      } else url = url.concat('-d');
+      // amazon-sec doesn't have yet documentation added
+      if (this.suite === 'amazon-sec') {
+        url = url.slice(0, -1);
+        return url;
+      }
+      const suiteNameBeforeDot = this.suite.split('.')[0];
+      if (browsertimeInteractiveTests.includes(suiteNameBeforeDot)) {
+        url = url.concat(suiteNameBeforeDot);
+        suffixForSuite = '-i';
+      } else {
+        url = url.concat(this.suite);
+        if (browsertimeBenchmarksTests.includes(this.suite)) {
+          suffixForSuite = '-b';
+        } else if (browsertimeCustomTests.includes(this.suite)) {
+          suffixForSuite = '-c';
+        } else if (this.platform.includes('android')) {
+          suffixForSuite = '-m';
+        } else suffixForSuite = '-d';
+      }
```

Comment:
Maybe we can move these lines for figuring out the suffix in a separate function.